### ISSUE TITLE
zeroclipboard cannot delegate event ?

### ISSUE THREAD

**Issue by jfw10973** [zeroclipboard/zeroclipboard#575]
Hi, again~
well, I read your wiki and found zeroclipboard cannot delegate event (exp: I new a zeroclipboard Object with a  never existed node, and just append that node. It won't works ).
Am I right or misunderstanding ?

so I now I find a way to do that by var client = new zeroclipboard(); and client.clip(node) every time after append it.
It works now.
Am I right or just a hack ?

I wanna do it better so I look into the code and found a param called "bubbleEvents", it seems that has no relation with delegate...

**Comment by JamesMGreene** [comment]
That's correct.  I put some effort into a jQuery wrapper for ZeroClipboard to help with this shortcoming (see https://github.com/zeroclipboard/jquery.zeroclipboard), though I never did much testing of it.  Feel free to try it out and file some issues if it isn't working.

Without using jQuery for event delegation, adding that concept into the ZeroClipboard codebase itself was simply a poor ROI: way too much code for not enough value.

**Comment by JamesMGreene** [comment]
**Note to self:** Should ensure that the documentation is clear about its lack of support for event delegation as it relates to `clip`ping future elements.

**Comment by pavankumarlpersonal** [comment]
Facing the same issue. This should be documented at the earliest. This is highly misleading
